create-release.js

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env node
+
+const { Octokit } = require('@octokit/rest');
+const inquirer = require('inquirer');
+
+const octokit = new Octokit({
+    auth: '<GENERATE FROM HERE: https://github.com/settings/tokens>',
+});
+
+async function fetchAppParameters() {
+  const questions = [
+    {
+        type: 'input',
+        name: 'name',
+        message: "What is your name?",
+        default: 'Jeff',
+      },
+    {  
+        type: 'list',
+        name: 'verson',
+        message: "How will you describe the release upgrade?",
```

Comment:
Can we put links here? Like, to the docs here maybe which are better than attempts I've made to explain semver: https://docs.npmjs.com/about-semantic-versioning

Reply:
```suggestion
        message: "How will you describe the release upgrade? Note changes or any breaking changes if it's a major release, following Semantic Versioning conventions: https://docs.npmjs.com/about-semantic-versioning",
```